GUI/src/main/java/gui/support/Constants.java

```diff
@@ -13,6 +15,17 @@
 import java.util.Objects;
 
 public class Constants extends support.Constants {
+    public static final UIComponentBuilder UI_COMPONENT_BUILDER_INSTANCE = UIComponentBuilder.getInstance();
+    public static final String BUTTON_RELEASED =
+            "-fx-text-fill: white;" +
+            "-fx-font-weight: Bold;" +
+            "-fx-background-color: linear-gradient(rgb(0, 53, 105) 20%, rgb(26, 21, 129) 65%, rgb(0, 0, 65) 100%);" +
+            "-fx-border-color: white;";
```

Comment:
@coderabbitai this code you has provided isn't work , it keeps giving me Css parsing error

Reply:
@coderabbitai the css code is not duplicated because those two css code are for dark theme only. For light theme, the css is located in possibly Button.css file. So, I don't think this is required.